package.json

```diff
@@ -150,12 +152,12 @@
     "cross-env": "^7.0.2",
     "css-loader": "^5.0.1",
     "css-minimizer-webpack-plugin": "^1.2.0",
-    "electron": "^19.0.7",
-    "electron-builder": "^23.1.0",
-    "electron-builder-notarize": "^1.5.0",
+    "electron": "^23.1.4",
```

Comment:
ok.You've updated electron version. Are you sure that we will not have any issues? is node 16 recommended to use with electron 23?

Reply:
It's needed for mas build.
Full regression In the scope of this story. Regression will show do we have any issues or not.

Reply:
right. but I don;t think that it is good practice to have different version, no?

Reply:
reverted version